tests/compliance_operator_v2_test.go

```diff
@@ -45,3 +47,49 @@ func TestComplianceV2ProfileGet(t *testing.T) {
 	}
 	assert.Greater(t, len(profile.Rules), 0, "failed to verify ocp4-cis profile contains any rules")
 }
+
+func TestComplianceV2ScheduleRescan(t *testing.T) {
+	conn := centralgrpc.GRPCConnectionToCentral(t)
+	client := v2.NewComplianceScanConfigurationServiceClient(conn)
+	integrationClient := v2.NewComplianceIntegrationServiceClient(conn)
+	resp, err := integrationClient.ListComplianceIntegrations(context.TODO(), &v2.RawQuery{Query: ""})
+	if err != nil {
+		t.Fatal(err)
+	}
+	clusterId := resp.Integrations[0].ClusterId
+
+	scanConfigName := "cis-scan-schedule"
+	sc := v2.ComplianceScanConfiguration{
+		ScanName: scanConfigName,
+		ScanConfig: &v2.BaseComplianceScanConfigurationSettings{
+			OneTimeScan: false,
+			Profiles:    []string{"ocp4-cis", "ocp4-cis-node"},
+			ScanSchedule: &v2.Schedule{
+				IntervalType: 3,
+				Hour:         0,
+				Minute:       0,
+			},
+			Description: "Scan schedule for CIS profiles to run daily.",
+		},
+		Clusters: []string{clusterId},
+	}
+	_, err = client.CreateComplianceScanConfiguration(context.TODO(), &sc)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// This is a workaround to get the ID
+	scanConfigs, err := client.ListComplianceScanConfigurations(context.TODO(), &v2.RawQuery{Query: ""})
+	scanConfig := scanConfigs.Configurations[0]
+
+	defer client.DeleteComplianceScanConfiguration(context.TODO(), &v2.ResourceByID{Id: scanConfig.GetId()})
+
+	waitForComplianceSuiteToComplete(t, scanConfig.ScanName, 2*time.Second, 5*time.Minute)
```

Comment:
Something in these last few lines is leaving an ugly trace in the test logs, but I was able to get a clean run locally:

```
main: 2024/01/08 22:21:24.425850 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in AGGREGATING phase
main: 2024/01/08 22:21:26.424181 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in AGGREGATING phase
main: 2024/01/08 22:21:28.424240 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in AGGREGATING phase
main: 2024/01/08 22:21:30.422949 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in AGGREGATING phase
main: 2024/01/08 22:21:32.427442 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in AGGREGATING phase
main: 2024/01/08 22:21:34.426439 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in AGGREGATING phase
main: 2024/01/08 22:21:36.424230 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in AGGREGATING phase
main: 2024/01/08 22:21:38.423617 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in AGGREGATING phase
main: 2024/01/08 22:21:40.426861 common_test.go:354: Info: ComplianceSuite cis-scan-schedule reached DONE phase
[controller-runtime] log.SetLogger(...) was never called; logs will not be displayed.
Detected at:
        >  goroutine 10 [running]:
        >  runtime/debug.Stack()
        >       /usr/lib/golang/src/runtime/debug/stack.go:24 +0x65
        >  sigs.k8s.io/controller-runtime/pkg/log.eventuallyFulfillRoot()
        >       /home/lbragstad/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.16.3/pkg/log/log.go:60 +0xcd
        >  sigs.k8s.io/controller-runtime/pkg/log.(*delegatingLogSink).WithName(0xc000d121c0, {0x2ccc1fe, 0x14})
        >       /home/lbragstad/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.16.3/pkg/log/deleg.go:147 +0x4c
        >  github.com/go-logr/logr.Logger.WithName({{0x31150e0, 0xc000d121c0}, 0x0}, {0x2ccc1fe?, 0x412746?})
        >       /home/lbragstad/go/pkg/mod/github.com/go-logr/logr@v1.4.1/logr.go:345 +0x46
        >  sigs.k8s.io/controller-runtime/pkg/client.newClient(0x7f66c5229208?, {0x0, 0xc0004593b0, {0x3116fc0, 0xc0006b7da0}, 0x0, {0x0, 0x0}, 0x0})
        >       /home/lbragstad/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.16.3/pkg/client/client.go:122 +0xff
        >  sigs.k8s.io/controller-runtime/pkg/client.New(0xc0006b7da0?, {0x0, 0xc0004593b0, {0x3116fc0, 0xc0006b7da0}, 0x0, {0x0, 0x0}, 0x0})
        >       /home/lbragstad/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.16.3/pkg/client/client.go:103 +0x85
        >  github.com/stackrox/rox/tests.createDynamicClient(0xc0010c9440?)
        >       /home/lbragstad/go/src/github.com/rhmdnd/stackrox/tests/common_test.go:308 +0x285
        >  github.com/stackrox/rox/tests.waitForComplianceSuiteToComplete(0xc00124f860, {0xc001306060, 0x11}, 0x0?, 0xc001363ea8?)
        >       /home/lbragstad/go/src/github.com/rhmdnd/stackrox/tests/common_test.go:335 +0x50
        >  github.com/stackrox/rox/tests.TestComplianceV2ScheduleRescan(0xc00124f860)
        >       /home/lbragstad/go/src/github.com/rhmdnd/stackrox/tests/compliance_operator_v2_test.go:95 +0x6b8
        >  testing.tRunner(0xc00124f860, 0x2e970b0)
        >       /usr/lib/golang/src/testing/testing.go:1576 +0x10b
        >  created by testing.(*T).Run
        >       /usr/lib/golang/src/testing/testing.go:1629 +0x3ea
main: 2024/01/08 22:21:40.494116 common_test.go:343: Info: Waiting for ComplianceSuite to reach DONE phase
main: 2024/01/08 22:21:42.874670 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in LAUNCHING phase
main: 2024/01/08 22:21:44.562188 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in LAUNCHING phase
main: 2024/01/08 22:21:46.560399 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in LAUNCHING phase
main: 2024/01/08 22:21:48.560020 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in LAUNCHING phase
main: 2024/01/08 22:21:50.560371 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in RUNNING phase
main: 2024/01/08 22:21:52.559785 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in RUNNING phase
main: 2024/01/08 22:21:54.561911 common_test.go:357: Info: ComplianceSuite cis-scan-schedule is in RUNNING phase
```

Reply:
Adjust the dynamic client so that it doesn't throw this up in the logs.